config/routes.rb

```diff
@@ -4,6 +4,7 @@
  scope '/admin' do
   resources :apartments
  end 
+ get 'apartments/download_ads' => 'apartments#download_ads'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.